packages/website/src/docs/button.mdx

```diff
@@ -1,18 +1,32 @@
 ---
 title: Button
 component: Button
+description: Useful to communicate actions to your users.
 ---
 
-```js live=true
+```js live
 <Button>New project</Button>
 ```
 
-### Color
+### children
 
-There are 4 color options to choose from.
+<Prop>ReactNode</Prop>
 
-```js live=true
-<Button variant="blue">
+The content to render inside the Button.
+
+### variant
+
+<Prop isOptional default="gray">
+  gray | blue | green | red
+</Prop>
+
+There are 6 variants to choose from.
```

Comment:
Says there's 6 variants, but type and example shows 4

Reply:
Done